Source/WebCore/Modules/webcodecs/WebCodecsVideoEncoder.cpp

```diff
@@ -56,40 +61,216 @@ WebCodecsVideoEncoder::~WebCodecsVideoEncoder()
 {
 }
 
-void WebCodecsVideoEncoder::isConfigSupported(WebCodecsVideoEncoderConfig&&, Ref<DeferredPromise>&& promise)
+static bool isValidEncoderConfig(const WebCodecsVideoEncoderConfig& config)
 {
-    // FIXME: Implement.
-    promise->reject(Exception { NotSupportedError, "Not implemented"_s });
+    // FIXME: Check codec more accurately.
+    if (!config.codec.startsWith("vp8"_s) && !config.codec.startsWith("vp09.00"_s))
+        return false;
```

Comment:
When you do fix this, it would be good to share it with the other code we have in WebKit that does similar comparisons.

Reply:
Will do in a follow-up.